contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -139,11 +139,10 @@ The following keys are on the secret data:
 * **token-secret**. As defined above.
 * **expiration**. After this time the token should be automatically deleted.  This is encoded as an absolute UTC time using RFC3339.
 * **usage-bootstrap-signing**. Set to `true` to indicate this token should be used for signing bootstrap configs.  If omitted or some other string, it defaults to `false`.
```

Comment:
This is for secret itself, not the UX for kubeadm.  If a usage is missing from the token secret, users should assume that usage isn't allowed.  I'll make it more clear in the text.